### PR TITLE
Create 4.swap-2-var-val

### PR DESCRIPTION
**Pull request by ramnandansah**
plz review

### CODE REVIEW

File: 4.swap-2-var-val
Comment (saurav0014) [inline 4.swap-2-var-val:1]: ok

**Review by saurav0014 — APPROVED** [review]

**Comment by saurav0014** [comment]
please send again

**Comment by ramnandansah** [comment]
okay changes

**Comment by ramnandansah** [comment]
changed

**Comment by saurav0014** [comment]
ok merged

**Pull request merged**